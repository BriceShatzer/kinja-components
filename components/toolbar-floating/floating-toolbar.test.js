import * as React from 'react';

import { shallow } from 'enzyme';

import FloatingToolbar from './floating-toolbar';
import ToolbarItem from '../toolbar-item';

const noop = () => {};

describe('Floating toolbar', () => {
	it('should render a list', () => {
		const wrapper = shallow(<FloatingToolbar>
			<ToolbarItem title="Test" icon="test" onClick={noop} />
		</FloatingToolbar>);
		expect(wrapper).toMatchSnapshot();
	});
});
