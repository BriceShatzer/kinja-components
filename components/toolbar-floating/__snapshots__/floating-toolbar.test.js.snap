// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Floating toolbar should render a list 1`] = `
<EnsureDefaultTheme>
  <floating-toolbar__ToolbarWrapper>
    <li
      key="Test/.0"
    >
      <ToolbarItem
        icon="test"
        onClick={[Function]}
        title="Test"
      />
    </li>
  </floating-toolbar__ToolbarWrapper>
</EnsureDefaultTheme>
`;
